debian/patches/016_nx-X11_install-location.debian.patch

```diff
@@ -6,16 +6,16 @@ Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
 Last-Update: 2012-12-31
```

Comment:
Don't forget to update the comment here, which is also... kind of wrong. We manually pull files out of the staging dir and library and binary directories differ. Not to mention that life is even more complicated due to multilib...